src/api/dataStoreApi/actions/getSubdomainsById.ts

```diff
@@ -0,0 +1,43 @@
+import { Maybe } from "../../../utilities";
+import { Domain } from "../../../types";
+import { DomainCollection } from "../types";
+import { makeApiCall } from "../../helpers";
+import { ethers } from "ethers";
+
+export const getSubdomainsById = async (
+  apiUri: string,
+  tokenId: string,
+  header: Record<string, string>
+): Promise<Domain[]> => {
+  let response: Maybe<DomainCollection>;
+
+  try {
+    response = await makeApiCall<DomainCollection>(
+      `${apiUri}domains/subdomains/${tokenId}`,
```

Comment:
make sure to add the request body we discussed to hide the _id
`{ "options": { "projection": "_id": -1 } }`